redisearch/result.py

```diff
@@ -45,9 +48,15 @@ def __init__(self, res, hascontent, duration=0, has_payload = False, with_scores
             except KeyError:
                 pass
 
-            doc = Document(id, score=score, payload=payload, **fields) if with_scores else Document(id, payload=payload, **fields)
+            try:
```

Comment:
@chayim  This is the change I made to Guy's comment (add jsonPath as property to the result)